quad.go

```diff
@@ -0,0 +1,159 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package quad provides numeric integration of single variable functions by quadrature.
+//
+package quad
```

Comment:
I'm easy on this - either is fine for me. I was thinking similar thoughts because of my involvement in the cayley db project where the `quad` is package for handling four word graph entities, but I didn't see a great deal of possibility for collision.